packages/create-itp-react-app/templates/app/.eslintrc

```diff
@@ -1,5 +1,6 @@
 {
     "extends": "./node_modules/@inthepocket/itp-react-scripts/dist/config/eslintrc.js",
+    "parser": "babel-eslint",
```

Comment:
why do we need to switch the parser here?

Reply:
The generated catalog code needs it to get through the linter.

More specifically, these errors:
```
.../.../design-docs/catalog/components/itp-rcc-autocomplete/index.js
  12:9  error  Parsing error: Unexpected token =

.../.../design-docs/catalog/index.js
  4:19  error  Parse errors in imported module './pages': Unexpected token import (18:49)  import/no-named-as-default
  4:19  error  Parse errors in imported module './pages': Unexpected token import (18:49)  import/no-named-as-default-member

.../.../design-docs/catalog/pages.js
  18:49  error  Parsing error: Unexpected token import
```

Reply:
OK!